server/mergin/sync/workspace.py

```diff
@@ -354,13 +354,9 @@ def project_access(self, project: Project) -> List[ProjectAccessDetail]:
 
     def server_editors_count(self) -> int:
         if Configuration.GLOBAL_ADMIN or Configuration.GLOBAL_WRITE:
-            return User.query.filter(
-                is_(User.username.ilike("deleted_%"), False) | User.active
-            ).count()
-        if Configuration.GLOBAL_READ:
             return User.query.filter(
                 is_(User.username.ilike("deleted_%"), False) | User.active,
-                User.is_admin.is_(True),
+                User.is_admin.is_(False),
```

Comment:
why False?

Reply:
It was just prototype. I thinks it's removed